.github/workflows/release.yml

```diff
@@ -24,3 +26,68 @@ jobs:
           goarch: ${{ matrix.goarch }}
           ldflags: "-s -w -X main.version=${{ github.ref_name }}"
           binary_name: "gql-lint"
+
+  lipo:
+    name: Universal Mac binary
+    runs-on: macos-12
+    needs: releases-matrix
+    env:
+      ARM_FILE: "gql-lint-${{ github.ref_name }}-darwin-arm64.tar.gz"
+      INTEL_FILE: "gql-lint-${{ github.ref_name }}-darwin-amd64.tar.gz"
+      UNIVERSAL_FILE: "gql-lint-${{ github.ref_name }}-darwin-universal.tar.gz"
+    steps:
+      - name: Print used variables
+        run: |
+          echo "Repo: ${{ github.repository }}"
+          echo "Release: ${{ github.ref_name }}"
+
+      - name: Download release amd64 asset
+        uses: dsaltares/fetch-gh-release-asset@1.1.0
+        with:
+          repo: ${{ github.repository }}
+          version: "tags/${{ github.ref_name }}"
+          file: ${{ env.INTEL_FILE }}
+
+      - name: Download release arm64 asset
+        uses: dsaltares/fetch-gh-release-asset@1.1.0
+        with:
+          repo: ${{ github.repository }}
+          version: "tags/${{ github.ref_name }}"
+          file: ${{ env.ARM_FILE }}
```

Comment:
The action says you can use regex patterns, it does, but also fails to generate the names, going one by one worked